src/jit/function-builder.cc

```diff
@@ -236,6 +268,8 @@ void FunctionBuilder::SetUpLocals(TR::IlBuilder* b, const uint8_t** pc, VirtualS
   set_zero->              IndexAt(pValueType_, stack_base_addr,
   set_zero->                      Load("i")),
   set_zero->              ConstInt64(0));
```

Comment:
Is this code for allocating space on the value stack still necessary? Given that we're handling locals entirely using the virtual stack now, it doesn't seem like we need to be doing this any more...